shell/test.sh

```diff
@@ -1,114 +1,193 @@
 #!/usr/bin/env bash
+# Runs tests for the current project.
+set -euo pipefail
```

Comment:
Enabled `e` (exit on error) `u` (error on undefined env vars) `o pipefail` (fail on pipe errors). It makes bash scripts much more resilient, and also a forcing function to document env vars :D